vision/unit_tests/test__gax.py

```diff
@@ -32,6 +32,47 @@ def test_ctor(self):
             api = self._make_one(client)
         self.assertIs(api._client, client)
 
+    def test_gapic_credentials(self):
+        from google.cloud.gapic.vision.v1.image_annotator_client import (
+            ImageAnnotatorClient)
+
+        from .test_client import _make_credentials
```

Comment:
Yeah, I was thinking a `utils` module in the future might be better.
In principle, I am probably more allergic to copy-paste than the import (they both stink), but since it is a two-line function, I will copy-paste.